Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -109,14 +110,40 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
       .map { _ in AppEnvironment.current.currentUser }
       .map(isNotNil)
 
+    let notChangingPaymentMethod = context.map { context in
+      context.isUpdating && context != .changePaymentMethod
+    }
+    .filter(isTrue)
+
+    let paymentMethodSelected = self.creditCardSelectedProperty.signal.skipNil()
```

Comment:
Nit: Since this gets passed into the CTA to indicate whether or not it is enabled, does the CTA disable ApplePay as well? Technically we could still run an ApplePay transaction before the payment method is selected (since in that case, you select a card via the ApplePay dialog), but I also don't think that's a big deal.